src/SFA.DAS.AssessorService.Web/Views/Shared/_ZenDeskWidget.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@using SFA.DAS.AssessorService.Web.Extensions
+
+@if (ViewBag.HideZenDeskWidget == null || ViewBag.HideZenDeskWidget == false)
```

Comment:
Where is `HideZenDeskWidget` being set? Couldn't the existence of ZenDeskLabel be used as the indicator instead?

Reply:
Removed HideZenDeskWidget , Its not required ... we need to display Help Widget all the pages